Anlab.Core/Extensions/StringExtensions.cs

```diff
@@ -13,5 +14,29 @@ public static string SafeToUpper(this string value)
 
             return value.ToUpper(CultureInfo.InvariantCulture);
         }
+
+        public static bool ClearOutSetupPrice(this string value)
+        {

```

Comment:
@jSylvestre you can do this in one line `return string.Equals("GRINDONLY", value, StringComparison.OrdinalIgnoreCase)`

Reply:
@srkirkland Good point, originally Dirk had told me two values, but now that there is only one that is better.